tests/accelerators/test_cpu.py

```diff
@@ -22,6 +22,10 @@ def test_restore_checkpoint_after_pre_setup_default():
     assert not plugin.restore_checkpoint_after_setup
 
 
+def test_availability():
+    assert CPUAccelerator.is_available
```

Comment:
```suggestion
    assert CPUAccelerator.is_available()
```
otherwise it's tautological :)